facilities.html

```diff
@@ -1,142 +1,135 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
-    <!-- FAVICON LINK -->
-    <link rel="icon" href="">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Octavian Hotel | Facilities</title>
-    <!-- LINK FOR BOOSTRAP STYLE SHEET,RE ADRESS TO SUIT THE DESIGN -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link href="https://fonts.googleapis.com/css?family=Manjari|Pacifico&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css">
-    <!-- <link rel="stylesheet" href="https://unpkg.com/flickity@2/dist/flickity.min.css"> -->
+    <title>Facilities Page</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    
+    <!--CSS link placed here-->
+    <link rel="stylesheet" type="text/css" href="facility.css">
```

Comment:
`<link rel="stylesheet" type="text/css" href="facility.css">`

correctly make the link to facility.css